packages/get-credentials/src/index.js

```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line import/prefer-default-export
+export { default as getCredentials } from './get-credentials'
+export { getCredentialsFromEnvironment } from './get-credentials'
+export { setCredentialsFromEnvFile } from './get-credentials'
```

Comment:
Maybe not `setCredentialsFromEnvFile` but `getCredentialsFromEnvironment` is usable for modules where you don't use env files. Think we can use `getCredentialsFromEnvironment` for the integration tests.